notifications.yaml

```diff
@@ -0,0 +1,268 @@
+              - $ref: '#/definitions/PaginationResponse'
+          examples: {}
+        '404':
+          description: No user found.
+      parameters: []
+  /messages:
+    post:
+      operationId: submitMessage
+      summary: Submit Message
+      description: Submits a message for delivery to a user.
+      parameters:
+        - name: message
+          in: body
+          schema:
+            $ref: '#/definitions/MessageDefinition'
+      responses:
+        '201':
+          description: Message accepted for delivery.
+          schema:
+            allOf:
+              - properties:
+                  id:
+                    type: string
+                    format: uuid
+                  status:
+                    $ref: '#/definitions/MessageStatus'
+                required:
+                  - id
+                  - status
+            description: ''
+            type: object
+          headers:
+            Location:
+              type: string
+              description: >-
+                Location (URL) of accepted message resource.
+
+                A GET request to this URL returns the message status and
+                details.
+definitions:
+  ProblemJson:
+    title: Problem Type
+    type: object
+    properties:
+      type:
+        type: string
+        format: uri
+        description: |
+          An absolute URI that identifies the problem type.  When dereferenced,
+          it SHOULD provide human-readable documentation for the problem type
+          (e.g., using HTML).
+        default: 'about:blank'
+        example: 'https://example.com/problem/constraint-violation'
+      title:
+        type: string
+        description: |
+          A short, summary of the problem type. Written in english and readable
+          for engineers (usually not suited for non technical stakeholders and
+          not localized); example: Service Unavailable
+      status:
+        type: integer
+        format: int32
+        description: >
+          The HTTP status code generated by the origin server for this
+          occurrence
+
+          of the problem.
+        minimum: 100
+        maximum: 600
+        exclusiveMaximum: true
+        example: 503
+      detail:
+        type: string
+        description: |
+          A human readable explanation specific to this occurrence of the
+          problem.
+        example: Connection to database timed out
+      instance:
+        type: string
+        format: uri
+        description: >-
+          An absolute URI that identifies the specific occurrence of the
+          problem.
+
+          It may or may not yield further information if dereferenced.
+  MessageContent:
+    title: MessageContent
+    type: object
+    properties:
+      body_short:
+        type: string
+        description: Message content (short version).
+        minLength: 3
+        maxLength: 100
+      body_long:
+        type: string
+        description: Message content (long version).
```

Comment:
A long version of the message, in plain text. The content of this field will be delivered to channels that don't have any limit in terms of content size (e.g. email, etc...). The long version of the message is optional and should be provided only in cases when a longer message (compared to the short version provided in `body_short`) gives more information to the user. This field should not be a duplicate of `body_short`.